src/jsutils/inspect.js

```diff
@@ -9,40 +9,102 @@
 
 import nodejsCustomInspectSymbol from './nodejsCustomInspectSymbol';
 
+const MAX_ARRAY_LENGTH = 10;
```

Comment:
Should these be customizeable values? I realize these match, for instance, the current `console.log` max depth. For now it is probably better to hard-code, as we can always open up a customizeable option in the future, whereas if we supply that option now we can't easily take it back.